src/wp-includes/script-loader.php

```diff
@@ -1843,6 +1843,26 @@ function wp_just_in_time_script_localization() {
 	);
 }
 
+
+/**
+ * Prints a loader script if there is text/template registered script.
+ */
+function wp_print_template_loader_script() {
+	$wp_scripts = wp_scripts();
+	if ( $wp_scripts->has_delayed_inline_script() ) {
+		$output = <<<JS
+let wpLoadAfterScripts = ( handle ) => {
+	let scripts = document.querySelectorAll(`[type="text/template"][data-wp-executes-after="\${handle}"]`);
+	scripts.forEach( (script) => { 
+		script.setAttribute("type","text/javascript"); 
```

Comment:
```suggestion
```

WP use `$this->type_attr` for script type. Please correct me if i miss anything.